tools/codegen/core/gen_static_metadata.py

```diff
@@ -594,8 +594,25 @@ def f(i, p=p):
 print >> H, '  } named;'
 print >> H, '} grpc_metadata_batch_callouts;'
 print >> H
-print >> H, '#define GRPC_BATCH_INDEX_OF(slice) \\'
-print >> H, '  (GRPC_IS_STATIC_METADATA_STRING((slice)) ? (reinterpret_cast<grpc_core::StaticSliceRefcount*>((slice).refcount)->index > static_cast<uint32_t>(GRPC_BATCH_CALLOUTS_COUNT) ?  GRPC_BATCH_CALLOUTS_COUNT : static_cast<grpc_metadata_batch_callouts_index>(reinterpret_cast<grpc_core::StaticSliceRefcount*>((slice).refcount)->index)) : GRPC_BATCH_CALLOUTS_COUNT)'
+
+batch_idx_of_hdr = '#define GRPC_BATCH_INDEX_OF(slice) \\'
+static_slice = 'GRPC_IS_STATIC_METADATA_STRING((slice))'
+slice2sref = '(slice).refcount'
```

Comment:
Let's rename these so a future reader can understand them:

sref -> slice_ref
2 -> to
scast -> static_cast
inval -> invalid

E.g., slice2sref -> slice_to_slice_ref

Reply:
Will do; those were my hacks to get things to fit in 80 char lines :P

Reply:
Done.

Reply:
Ah I see...do you think that my requested changes combined with the reformatting will make things less readable? As it stands, I had to do some mental work and looking around to figure out what the variable names meant, which is not great. But I'm not sure if the 80 char requirement combined with the rename will make things worse.

Forgot to add: rcast , like scast, probably doesn't need to be a variable.

Reply:
Never mind, just saw your changes and they look good.